src/systems/ec/implementations/react/packages/radio/src/RadioGroup.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import RadioButton from './RadioButton';
+
+const RadioGroup = ({
+  labelId,
+  items,
+  name,
+  helperText,
+  label,
+  hideLabel,
+  invalid,
+  invalidText,
+  binary,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-radio__group', {
+    'ecl-radio__group--invalid': invalid,
+    'ecl-radio__group--binary': binary,
+  });
+
+  return (
+    <div
+      {...props}
+      role="radiogroup"
+      aria-labelledby={labelId}
+      className={classNames}
+    >
+      {label && (
+        <div
+          className={classnames('ecl-form-label', {
+            'ecl-form-label--hidden': hideLabel,
+          })}
+          id={labelId}
+        >
+          {label}
+        </div>
+      )}
+
+      {helperText && (
+        <div className="ecl-radio__help ecl-help-block">{helperText}</div>
```

Comment:
Why not a p?
Why not using aria-describedby?
https://developer.paciellogroup.com/blog/2014/12/using-aria-describedby-to-provide-helpful-form-hints/

Reply:
we could use a `p` but I don't see the point of having an `aria-describedby` (it isn't described by anything, content is in the tag)

Reply:
or maybe you meant adding it to the label

Reply:
Yes the input AND the radio group itself, so that it's not just a floating <p> but we give it some meaning.